### DISCUSSION TITLE
日期组件支持单独导出日期面板，vue2的时候可以单独引入

### DISCUSSION THREAD

**Discussion by vela666** [element-plus/element-plus#15528] [Ideas]
类似naiveui  https://www.naiveui.com/zh-CN/os-theme/components/date-picker#panel.vue
![image](https://github.com/element-plus/element-plus/assets/69613314/54686322-53e3-45a9-b7e4-ccb6844fa2a3)

**Comment by Dsaquel** [comment]
DatePickerPanel released in `2.11.0`.
https://element-plus.org/en-US/component/date-picker-panel.html
